shared/src/commonMain/kotlin/com/adammcneilly/pwhl/mobile/shared/ui/components/PlayByPlayMap.kt

```diff
@@ -0,0 +1,117 @@
+package com.adammcneilly.pwhl.mobile.shared.ui.components
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.CornerRadius
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PaintingStyle.Companion.Stroke
+import androidx.compose.ui.graphics.drawscope.Fill
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.dp
+import com.adammcneilly.pwhl.mobile.shared.displaymodels.PlayByPlayEventDisplayModel
+import com.adammcneilly.pwhl.mobile.shared.ui.theme.PWHLColors
+
+private const val MAX_X = 600
+private const val MAX_Y = 300
```

Comment:
This is specific to hockeytech api. We should fix, can do in a different PR tho.